setup/security/authorization/index.md

```diff
@@ -68,17 +68,42 @@ accounts.
 ### Accounts
 In the dark ages (before Fiat), only accounts could be restricted. Because
 Clouddriver is the source of truth for accounts, Fiat reaches out to Clouddriver
-to gather the list of available accounts. To add access restrictions to an
-account, update your `halconfig` to include the `requiredGroupMembership` field.
+to gather the list of available accounts. 
+
+There are two types of access restrictions to an account, `READ` and `WRITE`. Users must have
+`READ` permissions of an account to view the account's cloud resources, and`WRITE` permissions to
+make changes to the resources. If there are multiple permissions for one type of access on an
+account, users must have matching roles to all of them to be granted that type of access.
```

Comment:
> If there are multiple permissions for one type of access on an account, users must have matching roles to all of them to be granted that type of access.

This statement is unclear to me. A user must have at least one of the listed roles in order to be granted access, but it sounds like this statement is saying one needs to have _all_ of the listed roles.